lib/libefi/rdwr_efi_windows.c

```diff
@@ -37,6 +37,7 @@
 #include <sys/dktp/fdisk.h>
 #include <sys/efi_partition.h>
 #include <sys/byteorder.h>
+#include <umem.h>
```

Comment:
@lundman is there another include I should be using?
I only did this to fix an error about UMEM_DEFAULT not being defined.